pkgs/development/python-modules/accelerate/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, packaging
+, psutil
+, pytorch
```

```suggestion
, torch
```
`pytorch` is just an alias